net/uspot/Makefile

```diff
@@ -0,0 +1,95 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=uspot
+PKG_RELEASE:=1
+
+PKG_LICENSE:=GPL-2.0
+PKG_MAINTAINER:=Thibaut VARÈNE <hacks@slashdirt.org>
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/f00b4r0/uspot.git
```

Comment:
I am not so sure if we should accept newly created packages, because it will be used by a few users, but it will be available for all routers, etc. So, it will end up to provide quantity over quality and it will use so many resources.

Reply:
This is the packages feed, and with uspot people can finally ditch the abomination that is CoovaChilli.

Reply:
The packages feed has been a mess for years, but you completely ignore my valid argument that it can replace CoovaChilli.

Reply:
If I will rephrase it:

1. Are you aware that we do have so many packages here, which are not maintained anymore here?
The reasons:
We do have lack of maintainers (here) - no new contributors
Upstream projects are dead - what we should do about it? Keep it here and act like nothing is happening?

== **Just look at PCRE2 transition.**

2. Usually new projects are like huray in the beginning and over the time, if there are no users or developers (changed priorities), etc. then it goes to maintenance phase and later, the project is archived / read - only.

3. What is the audience for this package? You said "ditch the Coovachilli" what you dont like it about? Why you raised it only in comment?

We are still adding new packages here without proper policy, that what I am repeating every single time here. Honestly, we can add every single package. Is that what we want? I don't think so. That's why we should vote for it like main repo does.

And I can keep asking more questions, if this is what you want. How many packages you reviewed recently or updated?

Reply:
1. Yes. But there's a truckload of packages that don't even make sense on OpenWrt's primary target (routers/APs). That is not true for this new submission.
2. So because some projects disappear, we should block all new package submissions? How is innovation supposed to happen?
3. People who want to use a captive portal, without the need for a an external kernel module (kmod-ipt-coova). Coova has been around for years, and all I heard about it is that it's unmaintainable because of that external kernel module requirement.

As for how many packages I've reviewed/updated recently: not many, because most of the packages make no sense to me to use on OpenWrt.